.circleci/images/chrome/stable/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # https://hub.docker.com/r/browserless/chrome/dockerfile/
-FROM circleci/node:lts-browsers
+FROM cimg/node:lts
```

Comment:
you are using `cimg/node:current` in other Dockerfile and , `cimg/node:lts` here - why?

Reply:
@makarandp0 The other ones are on beta / unstable and @charliesantos and I were looking at the [image tags](https://circleci.com/developer/images/image/cimg/node#browsers) from circleci and decided that it makes sense for stable to be the current stable [node version](https://nodejs.org/en/) which is `16.14.0`, while "current" would be the latest features `17.6.0`.